app/build.gradle

```diff
@@ -54,58 +51,62 @@ android {
     }
 
     sourceSets {
+        main { assets.srcDirs = ['src/main/assets'] }
+
         if (enable_plugin_kotlin) {
             main.java.srcDirs += 'src/main/kotlin'
         }
     }
 
-    compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
-    }
-
+    flavorDimensions "default"
     productFlavors {
         flavorDefault {
         }
         flavorGplay {
             buildConfigField "boolean", "IS_GPLAY_BUILD", "true"
         }
         flavorTest {
-            applicationId "net.gsantner.memetastic_test"
+            applicationIdSuffix ".test"
             versionCode = Integer.parseInt(new Date().format('yyMMdd'))
             versionName = new Date().format('yyMMdd')
+            resValue "string", "app_name", "MemeTESTic"
```

Comment:
Don't inject `app_name` ever as string ressource. Simple reason, F-Droid scanner doesn't recognize this, and even more it's more natural to use string ressources for this.

Reply:
Hmm alright, if you want it that way. Also F-Droid doesn't build the nightlies, so...

Reply:
I myself build nightlies with F-droid locally. Including but not limited to.